recipe/bld.bat

```diff
@@ -0,0 +1,4 @@
+nmake -f psi\msvc.mak GSROOTDIR=%LIBRARY_PREFIX% WIN64= MSVC_VERSION=%VS_MAJOR% DEVSTUDIO=
+if errorlevel 1 exit 1
+
```

Comment:
Doesn't seem to be an `install` target. Maybe copy the `bin`, `lib`, `Resourse` dirs like the installer does (https://ghostscript.com/doc/9.22/Install.htm#Install_Windows). I'd probably move the `.lib` files to `%LIBRARY_LIB%` and `.dll` and `.exe` files to `%LIBRARY_BIN%` too so it fits with normal conda behaviour....